<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetMemoryZeile</class>
 <widget class="QWidget" name="WidgetMemoryZeile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>20</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="AdresseOut">
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>20</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-right: 1px solid rgb(0,50,0);
border-bottom: 1px solid rgb(0,50,0);
background-color: rgb(100,100, 200);
color: rgb(170, 170, 255);
</string>
     </property>
     <property name="text">
      <string notr="true">ADR</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="EditValue_0">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>20</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px solid gray;
background-color: rgb(200, 200, 255);
</string>
     </property>
     <property name="text">
      <string notr="true">00</string>
     </property>
     <property name="maxLength">
      <number>2</number>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="EditValue_1">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>20</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px solid gray;
background-color: rgb(180,180, 255);
</string>
     </property>
     <property name="text">
      <string notr="true">01</string>
     </property>
     <property name="maxLength">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="EditValue_2">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>20</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px solid gray;
background-color: rgb(200, 200, 255);
</string>
     </property>
     <property name="text">
      <string notr="true">02</string>
     </property>
     <property name="maxLength">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="EditValue_3">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>20</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px solid gray;
background-color: rgb(180,180, 255);
</string>
     </property>
     <property name="text">
      <string notr="true">03</string>
     </property>
     <property name="maxLength">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="EditValue_4">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>20</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px solid gray;
background-color: rgb(200, 200, 255);
</string>
     </property>
     <property name="text">
      <string notr="true">04</string>
     </property>
     <property name="maxLength">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="EditValue_5">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>20</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px solid gray;
background-color: rgb(180,180, 255);
</string>
     </property>
     <property name="text">
      <string notr="true">05</string>
     </property>
     <property name="maxLength">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="EditValue_6">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>20</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px solid gray;
background-color: rgb(200, 200, 255);
</string>
     </property>
     <property name="text">
      <string notr="true">06</string>
     </property>
     <property name="maxLength">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="EditValue_7">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>20</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px solid gray;
background-color: rgb(180,180, 255);
</string>
     </property>
     <property name="text">
      <string notr="true">07</string>
     </property>
     <property name="maxLength">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="EditValue_8">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>20</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px solid gray;
background-color: rgb(200, 200, 255);
</string>
     </property>
     <property name="text">
      <string notr="true">08</string>
     </property>
     <property name="maxLength">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="EditValue_9">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>20</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px solid gray;
background-color: rgb(180,180, 255);
</string>
     </property>
     <property name="text">
      <string notr="true">09</string>
     </property>
     <property name="maxLength">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="EditValue_10">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>20</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px solid gray;
background-color: rgb(200, 200, 255);
</string>
     </property>
     <property name="text">
      <string notr="true">0A</string>
     </property>
     <property name="maxLength">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="EditValue_11">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>20</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px solid gray;
background-color: rgb(180,180, 255);
</string>
     </property>
     <property name="text">
      <string notr="true">0B</string>
     </property>
     <property name="maxLength">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="EditValue_12">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>20</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px solid gray;
background-color: rgb(200, 200, 255);
</string>
     </property>
     <property name="text">
      <string notr="true">0C</string>
     </property>
     <property name="maxLength">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="EditValue_13">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>20</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px solid gray;
background-color: rgb(180,180, 255);
</string>
     </property>
     <property name="text">
      <string notr="true">0D</string>
     </property>
     <property name="maxLength">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="EditValue_14">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>20</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px solid gray;
background-color: rgb(200, 200, 255);
</string>
     </property>
     <property name="text">
      <string notr="true">0E</string>
     </property>
     <property name="maxLength">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="EditValue_15">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>20</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px solid gray;
background-color: rgb(180,180, 255);
</string>
     </property>
     <property name="text">
      <string notr="true">0F</string>
     </property>
     <property name="maxLength">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="ReadOut">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>20</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-left: 1px solid rgb(0,50,0);
border-bottom: 1px solid rgb(0,50,0);
background-color: rgb(100,100, 200);
color: rgb(170, 170, 255);</string>
     </property>
     <property name="text">
      <string notr="true">Lesen</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="WriteOut">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>20</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-left: 1px solid rgb(0,50,0);
border-bottom: 1px solid rgb(0,50,0);
background-color: rgb(100,100, 200);
color: rgb(170, 170, 255);</string>
     </property>
     <property name="text">
      <string notr="true">Schreiben</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
